ansible/roles/ansible-tvault-contego-extension/templates/contego_service_s3.j2

```diff
@@ -4,6 +4,7 @@ Requires=tvault-object-store.service
 [Service]
 User={{TVAULT_CONTEGO_EXT_USER}}
 Group={{TVAULT_CONTEGO_EXT_USER}}
+LimitNOFILE=65536
```

Comment:
Please avoid hardcoding in j2 templates. This can be moved to vars file and same variable can be referred in both j2 templates.

Reply:
Added New Variable contego_service_ulimits_count